### ISSUE TITLE
Building issue with 'keops_version' vs. 'version' file

### ISSUE THREAD

**Issue by saulzar** [getkeops/keops#190]
When building from github source there seems to be a missing 'version' file however there is a 'keops_version'

```
(base) ➜  keops git:(master) python setup.py install
Traceback (most recent call last):
  File "setup.py", line 11, in <module>
    with open(os.path.join(here, "pykeops", "version"), encoding="utf-8") as v:
  File "/home/oliver/miniconda3/lib/python3.8/codecs.py", line 905, in open
    file = builtins.open(filename, mode, buffering)
FileNotFoundError: [Errno 2] No such file or directory: '/home/oliver/Downloads/keops/pykeops/version'

```

I just copied one to the other and it builds, but then get this error at runtime once installed.
`
FileNotFoundError: [Errno 2] No such file or directory: '/home/oliver/miniconda3/lib/python3.8/site-packages/pykeops-1.5-py3.8.egg/pykeops/keops_version'`

**Comment by joanglaunes** [comment]
Hello @saulzar ,
Sorry for replying very late ; I just did the fix now. The error comes from the fix for another issue (#178) ; I had changed filename "version" to "keops_version" and fixed `__init.py__` accordingly, but forgot to do the same for `setup.py`.

**Comment by joanglaunes** [comment]
I think this fixes the issue, so I am closing it now, but you can reopen if needed.
